sdk/search/Azure.Search/src/Generated/Models/AutocompleteRequest.cs

```diff
@@ -1,6 +1,8 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
+// <auto-generated/>
```

Comment:
Does this mean I can turn back on all the warnings?

Reply:
We can try. This would only work if analyzers disable themselves in generated code.